tests/by-util/test_tail.rs

```diff
@@ -358,6 +358,16 @@ fn test_positive_lines() {
         .stdout_is("c\nd\ne\n");
 }
 
+/// Test for reading all but the first NUM lines: `tail -3`.
+#[test]
+fn test_obsolete_syntax() {
+    new_ucmd!()
+        .args(&["-3"])
```

Comment:
please add a test when the file is too small (compared to the -X)

Reply:
Got it.